src/quo/components/text_combinations/page_top/view.cljs

```diff
@@ -32,18 +33,20 @@
          (format-counter counter-bottom))]])
 
 (defn- header
-  [{:keys        [title input counter-top counter-bottom]
+  [{:keys        [title title-accessibility-label input counter-top counter-bottom
+                  title-right title-right-props]
     avatar-props :avatar}]
-  [rn/view {:style style/header}
-   [rn/view {:style style/header-title}
-    (when avatar-props
-      [channel-avatar/view (assoc avatar-props :size :size-32)])
-    [text/text
-     {:weight :semi-bold
-      :size   :heading-1}
-     title]]
-   (when (= input :recovery-phrase)
-     [header-counter counter-top counter-bottom])])
+  (let [title-props (assoc title-right-props
+                           :title               title
+                           :right               title-right
+                           :accessibility-label title-accessibility-label)]
+    [rn/view {:style style/header}
+     [rn/view {:style style/header-title}
+      (when avatar-props
+        [channel-avatar/view (assoc avatar-props :size :size-32)])
+      [standard-title/view title-props]]
```

Comment:
The page-top component now supports parameters for its standard/title component. Needed because one screen uses a button there.